app/uk/gov/hmrc/nisp/views/main.scala.html

```diff
@@ -42,7 +42,7 @@
     hideBreadcrumb: Boolean = false)(mainContent: Html)(implicit request : Request[_], authContext: Option[AuthContext] = None, user: Option[NispUser] = None, partialRetriever: uk.gov.hmrc.play.partials.CachedStaticHtmlPartialRetriever)
 
 @linkElement = {
-    <link rel="stylesheet" href='@controllers.routes.Assets.at("stylesheets/nisp.css")'/>
+    <link rel="stylesheet" href='@routes.Assets.at("stylesheets/nisp.css")?version=@{ApplicationConfig.assetVersion}'/>
```

Comment:
You need to use `applicationConfig` which is the one that's passed into the method, rather than the global object.